src/app/forgot-password/forgotPassword.component.ts

```diff
@@ -1,19 +1,17 @@
 import {Component, OnInit} from '@angular/core';
-import {FormBuilder, Validators, ControlGroup} from '@angular/common';
 
 import {EmailValidator} from '../email-validator';
+import {FormBuilder, FormGroup, Validators, FormControl} from "@angular/forms";
```

Comment:
please move this back up. I like to group angular specific imports apart from application specific imports